geode-core/src/main/java/org/apache/geode/internal/statistics/platform/LinuxProcessStats.java

```diff
@@ -72,7 +72,7 @@ public static StatisticsType getType() {
   public static ProcessStats createProcessStats(final Statistics stats) { // TODO: was
                                                                           // package-protected
     if (stats instanceof LocalStatisticsImpl) {
-      HostStatHelper.refresh((LocalStatisticsImpl) stats);
+      OsStatisticsProvider.build().refresh((LocalStatisticsImpl) stats);
```

Comment:
Please change this. The only code that calls this createProcessStats method is in OsStatisticsProvider. So move this first block of code (the refresh call) to that caller. That way we will not need to build a new instance of OsStatisticsProvider each time createProcessStats is called.

Reply:
+1, Im fixing this